src/helpers/createFiles.js

```diff
@@ -0,0 +1,25 @@
+const { log } = require('cozy-konnector-libs')
+
+function createFiles(apps) {
+  log('info', 'createFiles starts')
+  let files = []
+  try {
+    for (const app of apps) {
+      files.push({
+        title: app.title[0],
+        description: app.description,
+        url: app.source,
+        thumbnail: app.vignette,
+        source: app.rights,
+        networkAccess: app.networkAccessibility,
+        hubMetadata: app.hubMetadata
+      })
+    }
+    return files
+  } catch (err) {
+    log('error', 'something wen wrong when creating files')
+    throw err
```

Comment:
There is nothing to catch here? Except if you have `title === undefined`  then you'll have `can't read [0] of undefined` but this is not relating to "creatingFiles".